submarine-server/server-core/src/main/java/org/apache/submarine/server/manager/ModelVersionManager.java

```diff
@@ -50,16 +50,14 @@ public class ModelVersionManager {
    *
    * @return object
    */
+  private static class ModelVersionManagerHolder {
+    private static ModelVersionManager manager = new ModelVersionManager(ModelVersionService.getInstance(),
+                                                                        new ModelVersionTagService(),
```

Comment:
new class param should also be avoided here.

Reply:
Hi @cdmikechen ,
Not sure what you mean by adding some test cases for the minio client section here.

Reply:
New code changes need to be supported by test cases, otherwise they will reduce code coverage and fail to prove code robustness.